core-services/release-controller/_repos/ocp-4.3-openstack-15.repo

```diff
@@ -0,0 +1,10 @@
+[openstack-15-rhel8]
+name = rhel-openstack
+baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.2/openstack-15-for-rhel-8-rpms
```

Comment:
need probably to create the mirror under 4.3 to be able to sync from there, but this should be fine for now